apollo-portal/src/main/java/com/ctrip/framework/apollo/openapi/v1/controller/AppController.java

```diff
@@ -84,7 +84,7 @@ public AppController(final PortalSettings portalSettings,
   @GetMapping("/apps")
   public List<OpenAppDTO> findApps(@RequestParam(value = "appIds", required = false) String appIds) {
     final List<App> apps = new ArrayList<>();
-    if (StringUtils.isEmpty(appIds)) {
+    if (StringUtils.hasLength(appIds)) {
```

Comment:
The logical result is reversed, which is wrong

```suggestion
    if (!StringUtils.hasLength(appIds)) {
```